packages/devtools-launchpad/stories/index.js

```diff
@@ -36,7 +36,8 @@ const getTabs = (tabs, state) => {
 
 const renderLandingPage = (props) => {
   return React.DOM.div({}, React.createElement(LandingPage, Object.assign({
-    onFilterChange: action("FILTER_TABS")
+    onFilterChange: action("FILTER_TABS"),
```

Comment:
was a bit puzzled by the inconsistency between FILTER_TABS and onTabClick. I suppose FILTER_TABS refers to the action, while onTabClick is never connected to any actual action.

From the LandingPage point of view, onFilterChange by itself is not linked to the FILTER_TABS action either. Maybe FILTER_TABS -> onFilterChange ?